cloudbuild.yaml

```diff
@@ -12,7 +12,7 @@ steps:
         "-f",
         "./Dockerfile",
       ]
-    timeout: 180s
+    timeout: 1200s
```

Comment:
this timeout is necessary for a full rebuild. Rust compilation is very slow. It should however be cached for almost all PRs